service_configuration_lib/spark_config.py

```diff
@@ -63,6 +63,12 @@
     'spark.kubernetes.executor.label.paasta.yelp.com/cluster',
 }
 K8S_AUTH_FOLDER = '/etc/spark_k8s_secrets'
+DEFAULT_SPARK_K8S_SECRET_VOLUME = {
+    'hostPath': '/etc/pki/spark',
+    'containerPath': K8S_AUTH_FOLDER,
```

Comment:
Is it possible to keep K8S_AUTH_FOLDER same as hostPath everywhere?
I personally prefer keeping things consistent (here hostPath == containerPath) as much as possible for simplicity (debugging, memorization, ease manual testing).